sources/api/ecs-settings-applier/src/main.rs

```diff
@@ -30,6 +30,9 @@ struct ECSConfig {
 
     #[serde(skip_serializing_if = "Option::is_none")]
     privileged_disabled: Option<bool>,
+
+    #[serde(skip_serializing_if = "std::vec::Vec::is_empty")]
+    available_logging_drivers: Vec<String>,
```

Comment:
Is there a complete set of known logging drivers, such that we can use an Enum rather than a string? Or does the ability to add logging drivers via plugin make that impossible?

Reply:
There's a complete set of built-in logging drivers, so we could do that.  But adding via plugin is also a supported scenario with ECS and making it available requires passing the appropriate value to the ECS agent.  We could decide to disallow plugins for Bottlerocket right now and add it later if that ends up being the preference; I don't have a strong opinion.

Reply:
+1 to keeping them as strings for the time being.

IIRC, if users were to load a log driver plugin at runtime (with a task or otherwise) then they may want to run other tasks using their driver (which has to be listed here).